crates/ruff_workspace/src/options.rs

```diff
@@ -2381,11 +2390,141 @@ impl PyUpgradeOptions {
     }
 }
 
+#[derive(Debug, PartialEq, Eq, Serialize, Deserialize)]
+#[serde(untagged)]
```

Comment:
Using `untagged` allows us to support both the old `SerializationFormat` and `Format` group. The only downside is that serde's error messages aren't any useful because it only tells you that what you have is neither of the two.